package.json

```diff
@@ -76,7 +76,7 @@
     "jest": "^26.6.3",
```

Comment:
The issue of having to place things on a light background is just temporary until we fully support dark-mode in lime-elements, right?

If so, then updating this last section of the _Contributing_ page probably isn't worth it, since we'll then probably just forget to change it back, right?

![image](https://user-images.githubusercontent.com/154725/118814767-c7c0fa80-b8b0-11eb-8806-4db0d462ecb4.png)